build/common.props

```diff
@@ -25,7 +25,7 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(SrcTargets)</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">$(SrcStandardTargets)</TargetFrameworks>
+    <TargetFramework Condition=" '$(OS)' != 'Windows_NT' ">$(SrcStandardTargets)</TargetFramework>
```

Comment:
Why this change?

Reply:
SrcStandardTargets only targets netstandard2.0 in the targets.props. Is it still necessary to use TargetFrameworks for a single target?

Reply:
I see, reverting back this change. Thanks Jean-Marc!